NuGet.config

```diff
@@ -15,6 +15,7 @@
     <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
     <add key="dotnet-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
     <add key="benchmark-dotnet-prerelease" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/benchmark-dotnet-prerelease/nuget/v3/index.json" />
+    <add key="bdn-temporary" value="https://ci.appveyor.com/nuget/benchmarkdotnet" />
```

Comment:
Instead of using this new package source, we need to push the package version we need into the `benchmark-dotnet-prerelease` feed.

Reply:
I have updated the package at `benchmark-dotnet-prerelease`, so we should be able to get rid of this line now.

Reply:
@DrewScoggins thank you! I've reverted the change.